### DISCUSSION TITLE
视频内新弹窗

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#4770] [综合讨论 / General]
感觉出现概率挺低的，几个月才会有一次，出现位置也不固定

![图片](https://github.com/the1812/Bilibili-Evolved/assets/81268244/32a1aff0-c4c3-4579-85cc-dc9b10ff81a7)

[HTML.txt](https://github.com/user-attachments/files/15507460/HTML.txt)

**Comment by the1812** [comment]
https://github.com/the1812/Bilibili-Evolved/blob/7800adbb02ef884e269ce82c9c8564a17662c279/registry/lib/components/video/player/remove-popup/remove-popup.scss#L41-L50

从你提供的 HTML 看元素的 class 是 `bili-qoeFeedback`, 但这个在三个月前就已经支持屏蔽了 (启用 `删除视频弹窗` 的 `评分` 类别)

**Reply by Tinhone** [reply]
原来如此，类别也确实合理，和普通弹窗的区别也许是这弹窗可能不是 UP 主设置的

**Comment by Geeyun-JY3** [comment]
今天在 Android 客户端碰到了个询问音量均衡效果如何的，好像还是投票类的。
所以这是 UP 主设的还是破站搞的。

**Reply by Tinhone** [reply]
感觉像是 B 站搞的，这弹窗无论是否互动，如果消失都是永久性的。出现的位置和时间也很随机，不像考虑了视频内容，在有些蓝闪电账号的视频里也会出现
